docs/src/content/en/docs/workflows-vnext/inngest-workflow.mdx

```diff
@@ -175,6 +189,7 @@ export const mastra = new Mastra({
   server: {
     host: '0.0.0.0',
```

Comment:
Let's add a comment to say why we need this, it's very important in fact especially for local dev with docker.